packages/framework/ai-collab/api-report/ai-collab.alpha.api.md

```diff
@@ -46,6 +47,13 @@ export function createMergableDiffSeries(diffs: Difference[]): Difference[];
 // @alpha
 export function createMergableIdDiffSeries(oldObject: unknown, diffs: Difference[], idAttributeName: string | number): Difference[];
 
+// @alpha
+export interface Diff {
```

Comment:
Just curious: is there information we don't want to expose in `TreeEdit`? Seems like we could just export that directly, unless there are implementation details we want to hide.

As it is, I don't think this information is enough to be generally useful, except for in the very specific use case of our example app.

Reply:
Although `TreeEdit` is not marked as `@internal`, it still has a lot of information we don't need to expose, like the node path in `SharedTree`.

Reply:
Hmmm...I think we can reuse `Difference` type, which is a @alpha